Library/Formula/x265.rb

```diff
@@ -13,20 +14,17 @@ class X265 < Formula
     sha256 "440805e02452b80400ff428967bd4c5d24f042bea2cc6b5d76a3a37648965a60" => :mountain_lion
   end
 
-  head 'https://bitbucket.org/multicoreware/x265', :using => :hg
+  option "with-16-bit", "Build a 16-bit x265 (default: 8-bit)"
 
-  depends_on 'yasm' => :build
-  depends_on 'cmake' => :build
-  depends_on :macos => :lion
+  deprecated_option "16-bit" => "with-16-bit"
```

Comment:
I don't think there's 16/32-bit options for this flag.  HIGH_BIT_DEPTH is somewhat experimental/cpu heavy, I don't think it should be enabled by default.  Above LGTM.